docs/backend/upgrading/version-specific-migration/upgrade-to-61.md

```diff
@@ -117,3 +117,85 @@ If you have an existing Plone 5.2 or 6.0 site and you migrate to 6.1, then migra
 -   If the `plone.app.discussion` Python package is _not_ in your setup, but the site has existing comments (discussions), then the migration code stops with an error.
     Apparently you _were_ using comments in your site.
     Add the `plone.app.discussion` package to your dependencies, and run the migration again.
+
+
+## Distributions
+
+Plone 6.1 introduces the concept of a Plone {term}`distribution`.
+A Plone distribution is a Python package that defines specific features, themes, add-ons, and configurations that get activated when creating a Plone site.
+Now it is available in core Plone as the recommended way for creating a Plone site.
+
+When you install the [`Plone` 6.1 Python package](https://pypi.org/project/Plone/), you get several new packages and two distributions:
```

Comment:
We need to reference how someone may install Plone 6.1. I would suggest something like this, since this is all we have at this moment. It's bad because it doesn't cover Classic UI.

```suggestion
When you [install the `Plone` 6.1 Python package](https://6.docs.plone.org/install/), you get several new packages and two distributions:
```

From there, we have [Create a project with Volto (development or pre-release)](https://6.docs.plone.org/install/create-project-cookieplone.html) (a very recent addition) but nothing for Classic UI for Plone 6.1. 😱 

----

As a side note, IMO, this is a critical problem that should block the release of Plone 6.1 final. Installation of Plone 6.1 and major development blockers—specifically how to write tests, create an add-on, and customize a theme—should be resolved. I don't know if anyone shares this opinion.